TestCases/flamelet/02_laminar_premixed_ch4_flame_hx_ad/lam_prem_ch4_hx_ad.cfg

```diff
@@ -128,7 +128,7 @@ MARKER_ANALYZE_AVERAGE= MASSFLUX
 OUTPUT_FILES= RESTART, RESTART_ASCII, PARAVIEW
 VOLUME_OUTPUT= RESIDUAL, PRIMITIVE
 VOLUME_ADJ_FILENAME= ch4_flame_hx_ad
-
+WRT_RESTART_COMPACT = NO
```

Comment:
Why are these needed to pass regressions? Are they using a filediff?

Reply:
Yes. Should I update the restart files instead?

Reply:
No this is fine.